src/api/app/controllers/webui/project_controller.rb

```diff
@@ -901,7 +901,7 @@ def set_maintained_project
 
   def load_project_info
     find_maintenance_infos
-    @ipackages = @project.linked_packages.map { |package| [ package.name, package.project.name ] }
+    @ipackages = @project.packages_from_linked_projects.map { |package| [ package.name, package.project.name ] }
```

Comment:
Given that packages_from_linked_projects is only used here, I would move the map to the other method and replace it by an SQL select.